libs/alcotest_ext/Alcotest_ext.ml

```diff
@@ -0,0 +1,129 @@
+(*
+   Utilities for writing test suites for Alcotest.
+
+   Keep in mind that some of this may become its own library or move to
+   Alcotest.
+*)
+
+open Printf
+
+type output = Stdout | Stderr | Merged_stdout_stderr | Separate_stdout_stderr
+
+type 'a t = {
+  category : string list;
+  name : string;
+  func : unit -> 'a;
+  (* Options *)
+  speed_level : Alcotest.speed_level;
+  check_output : output option;
+  (* Automatically determined *)
+  id : string;
+}
+
+type test = unit t
+type lwt_test = unit Lwt.t t
+
+(* Legacy type that doesn't support options *)
+type simple_test = string * (unit -> unit)
+
+(****************************************************************************)
+(* Helpers *)
+(****************************************************************************)
+(*
+   Dumb stuff for which
```

Comment:
for which ???

Reply:
🤔 it's probably a good idea that I didn't finish the sentence 😅. Something like the OCaml standard library should have had these functions 25 years ago.